README.md

```diff
@@ -1,3 +1,41 @@
 # Unfug für Anfänger
```

Comment:
You need a `\frontmatter` here _or_ you could add another file with just `\frontmatter` in it and name the link to this file appropriately so the `\frontmatter` appears to be before the readme contents.